tests/test-dirs/config/dot-merlin-reader/load-config.t

```diff
@@ -3,14 +3,16 @@
   > S source/dir
   > BH build-hidden/dir
   > SH source-hidden/dir
+  > STDLIB /stdlib
   > EOF
 
   $ FILE=$(pwd)/test.ml; dot-merlin-reader <<EOF | sed 's#[0-9]*:#?:#g'
   > (4:File${#FILE}:$FILE)
   > EOF
-  ((?:B?:$TESTCASE_ROOT/build/dir)(?:S?:$TESTCASE_ROOT/source/dir)(?:BH?:$TESTCASE_ROOT/build-hidden/dir)(?:SH?:$TESTCASE_ROOT/source-hidden/dir))
+  ((?:B?:$TESTCASE_ROOT/build/dir)(?:S?:$TESTCASE_ROOT/source/dir)(?:BH?:$TESTCASE_ROOT/build-hidden/dir)(?:SH?:$TESTCASE_ROOT/source-hidden/dir)(?:STDLIB?:/stdlib))
 
-  $ echo | $MERLIN single dump-configuration -filename test.ml 2> /dev/null | jq '.value.merlin'
+Use ocamlmerlin instead of $MERLIN for this test because $MERLIN configures the stdlib
```

Comment:
This comment is mystifying. It should also say "... and we want to observe that the stdlib can be configured via STDLIB in this test."